.github/workflows/ci.yml

```diff
@@ -1,23 +1,38 @@
-name: GitHub Actions
-on: [push, pull_request]
+name: Build, test, and lint
+on:
+  pull_request:
+  push:
+    branches: [master]
+  release:
+    types: [published]
 jobs:
-
   build:
     name: Build
     runs-on: ubuntu-latest
     steps:
-
-    - name: Set up Go 1.13
-      uses: actions/setup-go@v1
-      with:
-        go-version: 1.13
-      id: go
+    - name: Set up Go
+      uses: actions/setup-go@v2
 
     - name: Check out code
-      uses: actions/checkout@v1
+      uses: actions/checkout@v2
 
-    - name: Build, Test, Lint
-      run: |
-        export GOPATH=$HOME/go && export GOBIN=$(go env GOPATH)/bin && export PATH=$PATH:$GOPATH&& export PATH=$PATH:$GOBIN && mkdir -p $GOBIN
```

Comment:
The reason this was on a single line was because it is completely uninteresting, and it's a side effect of us still doing a `go get` somewhere for some tools, which we should remove.

But if you think it's better to have it expanded each on its, line, I'm fine with it as well.